xla/debug_options_flags.cc

```diff
@@ -189,7 +189,12 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_partitioning_algorithm(
       DebugOptions::PARTITIONING_ALGORITHM_NOOP);
 
+#ifdef TENSORFLOW_USE_ROCM
```

Comment:
We are actively removing `#ifdef` from XLA's code-base and also avoid new ones wherever possible.

Instead of setting this here, please adjust the code where the flag is used:
https://github.com/openxla/xla/blob/main/xla/service/gpu/gpu_compiler.cc#L1455-L1458